Sources/BraveWallet/Preview Content/MockJsonRpcService.swift

```diff
@@ -358,6 +358,34 @@ extension BraveWallet.NetworkInfo {
     supportedKeyrings: [BraveWallet.KeyringId.solana.rawValue].map(NSNumber.init(value:)),
     isEip1559: false
   )
+  static let mockFilecoinMainnet: BraveWallet.NetworkInfo = .init(
+    chainId: BraveWallet.FilecoinMainnet,
+    chainName: "Filecoin Mainnet",
+    blockExplorerUrls: [""],
+    iconUrls: [],
+    activeRpcEndpointIndex: 0,
+    rpcEndpoints: [URL(string: "https://rpc.ankr.com/filecoin")!],
```

Comment:
What's the purpose of these mock classes? Also, do we proxy requests to the filecoin RPC?

Reply:
Mock classes are for SwfitUI preview and unit tests. 
All rpc requests are proxied in core. this rpc end point is a way to customize the rpc end point for custom networks.